central/scannerdefinitions/handler/handler.go

```diff
@@ -38,24 +39,31 @@ import (
 )
 
 const (
-	definitionsBaseDir = "scannerdefinitions"
+	// rootDir is the root directory in which all Scanner data is written.
+	rootDir = "scannerdefinitions"
```

Comment:
This is the pattern used to create a temporary directory. I think it should be called:

```suggestion
	// tmpDirPattern is the pattern for the directory in which all Scanner data is written.
	tmpDirPattern = "scannerdefinitions-*"
```

The `-*` will create a nicer name like `/tmp/scannerdefinitions-123456789`, and make it obvious that it is a pattern (although the `*` at the end is not necessary).